integration/configs/app_quit_video_test.config.json

```diff
@@ -10,7 +10,7 @@
       "noOfThreads": 1,
       "testImpl": "AppQuitVideoTest.js",
       "payload": {
-        "url": "http://localhost:8080/",
+        "url": "http://localhost:8080/?earlyConnect=1",
```

Comment:
Is there any specific reason we enable this for these particular tests but not the others?

Reply:
See earlier commentary — it makes sure we get some coverage on both paths without having to do something redundant and costly like running the entire suite twice.

Reply:
Got it. I understand the reasoning just wonder if there is any specific reason to choose these specific tests :)

Reply:
Nope, no reason. I just picked a few at random.